codecov/__version__.py

```diff
@@ -0,0 +1,8 @@
+__title__ = 'codecov'
```

Comment:
nit: alpha this please

Reply:
I'm not sure we want to change the package name, won't that break everyone's import?

Reply:
errrr, I mean
```
  1 __author__ = 'Codecov'
  2 __author_email__ = 'support@codecov.io'
  3 __copyright__ = 'Copyright 2020 Codecov'
  4 __description__ = 'Hosted coverage reports for GitHub, Bitbucket and Gitlab'
  5 __license__ = 'Apache 2.0'
  6 __title__ = 'codecov'
  7 __url__ = 'https://github.com/codecov/codecov-python'
  8 __version__ = '2.1.3'
```